examples/templates/example.html

```diff
@@ -18,7 +18,7 @@
   </head>
   <body>
     <header class="navbar navbar-expand-md navbar-dark mb-3 py-0 fixed-top" role="navigation">
-      <a href='/' class='navbar-brand'><img src="/theme/img/logo-dark.svg" width="70px" height="70px" alt="">&nbsp;OpenLayers</a>
+      <a href='/' class='navbar-brand'><img src="/theme/img/logo-dark.svg" width="70" height="70" alt="">&nbsp;OpenLayers</a>
```

Comment:
Thanks, @marcjansen.  I agree with @MoonE that the other instances should be changed as well.